concent_api/middleman/middleman_server.py

```diff
@@ -165,6 +167,7 @@ def _close_middleman(self) -> None:
             raise
 
         finally:
+            logger.info(f"Canceling tasks for connection with Concent. Number of tasks to cancel: {str(len(tasks))}.")
```

Comment:
You don't have to change the result of `len()` to `str`. It will work without it.